frontend/src/pages/profile/ProfileRatingsPage.tsx

```diff
@@ -1,7 +1,36 @@
+import { Box, Group, Paper, Title } from "@mantine/core";
+import { useQuery } from "@tanstack/react-query";
+import { useParams } from "react-router-dom";
+import { getAllRatingsByUserId } from "../../api/ratings";
+import { RatingChip } from "../../components/ratings/RatingChip";
+import MovieInfo from "../../components/reviews/MoiveInfo";
+
 export const ProfileRatingsPage = () => {
+  const { id } = useParams();
+
+  const { data: ratings } = useQuery(["ratings", { userId: id }], () =>
+    getAllRatingsByUserId(id)
+  );
+
+
+  if (!ratings) {
+    return <div>not found</div>;
+  }
+
   return (
     <div>
-      <div>Profile ratings page</div>
+      {ratings.map((rating, key) => {
+        return <Paper key={key} p="md" m="md">
+          
+          <MovieInfo id={rating.movieId} />
```

Comment:
MovieInfo is meant to be an entire page, I would take the markup for it and shrink it into its own component to show context for the rating